apps/web/utils/supabase/server.ts

```
@@ -1,11 +1,98 @@
+                  },
+                  error: null,
+                }),
+                single: async () => ({
+                  data: {
+                    id: "mock-id",
+                    slug: "test-workspace",
+                    name: "Test Workspace",
+                  },
+                  error: null,
+                }),
+                then: async () => [
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)